src/renderer/main.js

```diff
@@ -1,6 +1,6 @@
-/*global React*/
 
 import insight from './utils/insight';
+import ReactDOM from 'react-dom';
```

Comment:
NABD, but this could be a use case for ES6 destructuring

```js
import {render} from 'react-dom'

...

render(<Root />, document.getElementById('content'));
```